core/src/trezor/messages.py

```
@@ -7836,6 +7838,44 @@
+            self,
+            *,
+            message: "bytes",
+            address_n: "list[int] | None" = None,
+            message_version: "SolanaMessageVersion | None" = None,
+            message_format: "SolanaMessageFormat | None" = None,
+            application_domain: "bytes | None" = None,
+        ) -> None:
+            pass
+
+        @classmethod
+        def is_type_of(cls, msg: Any) -> TypeGuard["SolanaSignMessage"]:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `msg`

(ANN401)